pkg/fileutil/files.go

```diff
@@ -236,3 +240,68 @@ func CreateFile(p string) (*os.File, error) {
 	}
 	return os.Create(p)
 }
+
+func UploadFile(filePath, targetURL, formFileFieldName string) error {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return fmt.Errorf("error opening file: %w", err)
+	}
+	fmt.Println("File opened successfully")
```

Comment:
Is this a debug print or do we actually wanna display this to the user?

Reply:
Removed :)

Reply:
I dont see the debug print removed...